nixos/common.nix

```diff
@@ -43,22 +43,32 @@ let
 
 in {
   options.home-manager = {
-    useUserPackages = mkEnableOption ''
-      installation of user packages through the
-      <option>users.users.&lt;name&gt;.packages</option> option
```

Comment:
Could just move the final `''` to the end of the last line:

```
<option>users.users.&lt;name&gt;.packages</option> option''
```

Reply:
Sure, done.